prac_05/wimbledon.py

```diff
@@ -0,0 +1,47 @@
+"""
+CP1404 - Prac 5
+Program: Wimbledon
+"""
+
+FILENAME = 'wimbledon.csv'
+COUNTRY_INDEX = 1
+CHAMPION_INDEX = 2
+
+def main():
+    """Load records from file and display details of champions."""
+    records = load_records()
+    champion_to_win_count, countries = process_records(records)
+    display_results(champion_to_win_count, countries)
+
+def load_records():
```

Comment:
Consider adding error handling (e.g., a try-except block) when opening the file.